stats/cloud/api.go

```diff
@@ -112,3 +161,22 @@ func (c *Client) TestFinished(referenceID string, thresholds ThresholdResult, ta
 	err = c.Do(req, nil)
 	return err
 }
+
+func (c *Client) ValidateOptions(options lib.Options) error {
+	url := fmt.Sprintf("%s/validate-options", c.baseURL)
+
+	data := struct {
+		Options lib.Options `json:"options"`
+	}{
+		options,
+	}
+
+	req, err := c.NewRequest("POST", url, data)
+	if err != nil {
+		return err
+	}
+
+	err = c.Do(req, nil)
```

Comment:
Can be simplified to: `return c.Do(req, nil)`. (I thought the linter was supposed to catch this…)